project/Dependencies.scala

```diff
@@ -2,12 +2,12 @@ import org.scalajs.sbtplugin.ScalaJSPlugin.autoImport._
 import sbt._
 
 object Dependencies {
-
+  // scalastyle:off
   //libs for testing
   lazy val testing: Def.Initialize[Seq[ModuleID]] = Def.setting(Seq(
-    "org.scalatest" %%% "scalatest" % Versions.scalaTest % Test,
+    "org.scalatest" %%% "scalatest" % Versions.scalaTest % Test/*,
 
-    "org.scalatest" %%% "scalatest-matchers" % Versions.scalaTest % Test))
+    "org.scalatest" %%% "scalatest-matchers" % Versions.scalaTest % Test*/))
```

Comment:
scalatest-matchers and scalatest right now have different versions. For scalatest the latest one is 	3.0.0-M16-SNAP4